x-pack/plugins/apm/public/components/app/transaction_details/waterfall_with_summary/waterfall_container/Waterfall/accordion_waterfall.tsx

```diff
@@ -154,3 +157,40 @@ export function AccordionWaterfall(props: AccordionWaterfallProps) {
     </StyledAccordion>
   );
 }
+
+function ToggleAccordionButton({
+  show,
+  isOpen,
+  childrenAmount,
+  onClick,
+}: {
+  show: boolean;
+  isOpen: boolean;
+  childrenAmount: number;
+  onClick: () => void;
+}) {
+  if (!show) {
+    return null;
+  }
+
+  return (
+    <div style={{ height: ACCORDION_HEIGHT, display: 'flex' }}>
+      <EuiFlexGroup gutterSize="xs" alignItems="center" justifyContent="center">
+        <EuiFlexItem grow={false}>
+          {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events */}
+          <div
```

Comment:
Why is this a div? If it were a button could we avoid the a11y problem? Could we use `EuiButtonIcon` instead of the `EuiIcon` wrapped in the div with the onclick?

Reply:
I cannot use a button here because it's already inside the accordion button.